WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -390,12 +390,12 @@ private extension HubMenu {
             comment: "Title of the view containing a single Product Review"
         )
         static let reviews = NSLocalizedString(
-            "hubMenu.coupons",
+            "hubMenu.reviewsList",
             value: "Reviews",
             comment: "Title of the view containing Reviews list"
         )
         static let coupons = NSLocalizedString(
-            "hubMenu.coupons",
+            "hubMenu.couponsList",
```

Comment:
> This PR fixed that so that both keys are now unique. I found that simply renaming one of the keys did not do the job, so instead both keys are renamed to be something new.

This is the right approach, but to make it show up right, only one change is strictly needed... unfortunately it's the wrong one in this case.

If you do the following, it works fine:

```
    static let reviews = NSLocalizedString(
            "hubMenu.coupons",
            value: "Reviews",
            comment: "Title of the view containing Reviews list"
        )
        static let coupons = NSLocalizedString(
            "hubMenu.couponsList",
            value: "Coupons",
            comment: "Title of the view containing Coupons list"
        )
```

Which would be fine because `hubMenu.coupons` **should** have a correct translation for the value `Reviews` in each language, even though the key is misleading.

That said... this is simply how Xcode handles the "translation" of the English value for the key – `genstrings` could behave differently, so I think it's good that you've replaced both keys.

Reply:
> This is the right approach, but to make it show up right, only one change is strictly needed... unfortunately it's the wrong one in this case.

That's a good point, also about using `couponList` only, with the trade-off of having the wrong key name for Reviews.

I had in mind your previous mention about translation and treating string as immutable, so it felt like it's just the safest step to rename both of them. Thanks for chiming in here!